Quadriga/src/main/webapp/WEB-INF/views/transformation/emptyTransformation.jsp

```diff
@@ -0,0 +1,5 @@
+<h1>Transformation is empty</h1>
+
+<p>
+There is nothing to download.
```

Comment:
We are sorry, but there is nothing to download. You either haven't started a transformation, the transformation process hasn't finished yet, or the transformation process failed.